data/json/itemgroups/guns.json

```diff
@@ -133,10 +133,11 @@
     "id" : "guns_rifle_common",
     "//" : "Rifles commonly owned by citizens and found in many locations.",
     "items": [
-      [ "browning_blr", 25 ],
-      [ "garand", 65 ],
       [ "ar15", 30 ],
+      [ "browning_blr", 25 ],
+      [ "cane_rifle", 20 ],
```

Comment:
Well at least take the spawn rate down. Its currently higher than the modernized mosin.